### DISCUSSION TITLE
tutorials/troubleshooting

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#93] [User-contributed notes]
# tutorials/troubleshooting

This page lists common issues encountered when using Godot and possible solutions. The editor runs slowly and uses all my CPU and GPU resources, making my computer noisy: This is a known issue, esp...

https://docs.godotengine.org/en/stable/tutorials/troubleshooting.html

**Comment by tech2go-web** [comment]
Whenever I open a project, the engine starts to close and write this error message on my console. How can I fix this?

Godot Engine v4.3.stable.official.77dcf97d8 - https://godotengine.org
OpenGL API 3.3.0 - Build 32.0.101.6129 - Compatibility - Using Device: Intel - Intel(R) UHD Graphics

Editing project: D:/new-game-project
Godot Engine v4.3.stable.official.77dcf97d8 - https://godotengine.org
ERROR: Condition "err != VK_SUCCESS" is true. Returning: ERR_CANT_CREATE
   at: _initialize_device (drivers/vulkan/rendering_device_driver_vulkan.cpp:972)
ERROR: Condition "err != OK" is true. Returning: err
   at: initialize (drivers/vulkan/rendering_device_driver_vulkan.cpp:1205)
ERROR: Failed to initialize driver for device.
   at: (servers/rendering/rendering_device.cpp:5372)
ERROR:             vkEndCommandBuffer: Invalid commandBuffer [VUID-vkEndCommandBuffer-commandBuffer-parameter]

**Reply by Calinou** [reply]
Please ask support questions on other community channels: https://godotengine.org/community/
